### ISSUE TITLE
update npm

### ISSUE THREAD

**Issue by ghost** [awernick/google-assistant-node#3]
Any plans updating the package on npm?

**Comment by awernick** [comment]
Yes. I'm trying to improve the library a bit to make it easier for general usage. I will update the package in a couple of days once I verify and test the new features.
